PocketKit/Sources/PocketKit/MyList/ItemsList/ActivityViewController.swift

```diff
@@ -0,0 +1,14 @@
+import SwiftUI
+
+struct ShareSheetView: UIViewControllerRepresentable {
+    let activity: PocketActivity
+
+    func makeUIViewController(context: Context) -> UIActivityViewController {
+        let controller = UIActivityViewController(activity: activity)
+        controller.sheetPresentationController?.detents = [.medium()]
```

Comment:
does not seem to make any difference in resolving the iOS 15 issue where the share sheet takes up the whole screen